ncard_app/forms.py

```diff
@@ -16,38 +17,76 @@ def __init__(self, *args, **kwargs):
+                        Column('given_name', css_class='col-md-4 mb-0'),
+                        Column('middle_name', css_class='col-md-3 mb-0'),
+                        Column('surname', css_class='col-md-4 mb-0'),
+                        css_class='row'
+                    ),
+                    Row(
+                        Column('email', css_class='row'),
+                        Column('email2', '', css_class='row'),
+                        css_class='row'
+                    ),
+                    Row(
+                        Column('phone_office', css_class='row'),
+                        Column('phone_mobile', css_class='row'),
+                        Column('phone_home', css_class='row'),
+                        css_class='row'
+                    ),
+                ),
+                
+                Tab('NCARD Info',
+                    HTML("<br>"),
+                    Row(
+                        Column('cre_role', css_class='col-md-4 mb-0'),
+                        Column('ncard_relation', css_class='col-md-4 mb-0'),
+                        Column('project', css_class='col-md-4 mb-0'),
+                        css_class='row'
+                    ),
+                    Row(
+                        Column('display_on_website', css_class='col-md-3 mb-0'),
+                        Column('profile_url', css_class='col-md-9 mb-0'),
+                        css_class='row'
+                    ),
+                ),
+
+                Tab('Individal Reseracher Details',
+                    HTML("<br>"),
+                    Row(
+                        Column('clinician', css_class='col-md-2 mb-0'),
+                        Column('orcid_id', css_class='col-md-3 mb-0'),
+                        Column('scopus_id', css_class='col-md-3 mb-0'),
+                        Column('wos_researcher_id', css_class='col-md-3 mb-0'),
+                        css_class='row'
+                    ),
+                    Row(
+                        Column('google_scholar', css_class='col-md-4 mb-0'),
+                        Column('researchgate', css_class='col-md-4 mb-0'),
+                        Column('loop_profile', css_class='col-md-4 mb-0'),
+                        css_class='row'
+                    ),
+                    Row(
+                        Column('linkedin', css_class='col-md-4 mb-0'),
+                        Column('twitter', css_class='col-md-3 mb-0'),
+                        Column('research_focus', css_class='col-md-5 mb-0'),
+                        css_class='row'
+                    ),
+                ),
+
+                Tab('Organisational Info',
+                    HTML("<br>"),
+                    Row('employers', css_class='row'),
+                    Row('location', css_class='row'),
+                    Row('organisation_primary', css_class='row'),
+                    Row('organisation_other', css_class='row'),
+                    Row('notes', css_class='row'),
```

Comment:
The "Notes" field can be about anything, not just the person's organisation. I think it should be put in the Contact Details tab, or completely outside the tab view.

Reply:
Added a new tab "Others" and put "note" into it

Reply:
If the notes are the only thing in the "Others" tab, a better name for the tab would be "Notes".